src/main/resources/application.yml

```diff
@@ -43,5 +43,5 @@ app:
 logging:
   file: orlp.log
   level:
-    com.softserve.academy.spaced.repetition: DEBUG
```

Comment:
Please also increase unit test log level as following https://www.mkyong.com/spring-boot/spring-boot-test-how-to-stop-debug-logs/